lib/puppet/network/http/request.rb

```diff
@@ -19,41 +19,42 @@ def route_into(prefix)
     self.class.new(headers, params, method, path, routing_path.sub(prefix, ''), client_cert, body)
   end
 
-  def format
+  def formatter
     if header = headers['content-type']
       header.gsub!(/\s*;.*$/,'') # strip any charset
       format = Puppet::Network::FormatHandler.mime(header)
-      if format.nil?
-        #TRANSLATORS "mime-type" is a keyword and should not be translated
-        raise _("Client sent a mime-type (%{header}) that doesn't correspond to a format we support") % { header: header }
-      else
-        assert_supported_format(format)
-        return format.name.to_s if format.suitable?
-      end
+
+      return valid_network_format?(format) ? format : nil
```

Comment:
Thoughts on renaming the local `format` variable here to something else, to avoid possible confusion/collision with the `format` method defined later?